files/mailer.rb

```diff
@@ -99,12 +99,12 @@ def handle
     begin
       timeout 10 do
         Mail.deliver do
-          to      mail_to
-          from    mail_from
-          subject subject
-          body    body
+          to       mail_to
+          from     mail_from
+          subject  subject
+          body     body
+          reply_to mail_to
```

Comment:
what will happen when mail_to is multiple email addresses? will mail servers in the path allow it? tbh I doubt it, I think it violates RFC and the message will be dropped

Reply:
Hmm. Not sure.. 
I guess I should test it :)